src/main/resources/application.properties

```diff
@@ -160,20 +160,20 @@ logging.logback.rollingpolicy.max-file-size=20KB
 logging.logback.rollingpolicy.max-history=1
 
 ### SMS Gateway URL details
-sms-username=<Enter SMS username>
-sms-password=<Enter SMS password>
-send-message-url=<Enter SMS sending URL here>
+sms-username=amrit
+sms-password=Amrit@321
```

Comment:
@swetatech24 Replace it with the placeholder. Do not put credentials in properties file.